spark/dl/src/main/scala/com/intel/analytics/bigdl/python/api/PythonBigDL.scala

```diff
@@ -55,12 +55,13 @@ object PythonBigDL {
 
   def ofDouble(): PythonBigDL[Double] = doubleInstance
 
-  def getInitMethod(initMethod: String): InitializationMethod = {
+  def getInitMethod(initMethod: String): Initializer = {
```

Comment:
We can have a object Initializer{} with multiple pre-defined init-methods, and can match by name for now.